src/replicator.rs

```diff
@@ -42,6 +43,9 @@ pub struct Replicator {
     fetch_stage: BlobFetchStage,
     t_window: JoinHandle<()>,
     pub retransmit_receiver: BlobReceiver,
+    // Currently unused. Stored to prevent the window service from experiencing errors when sending
+    #[allow(dead_code)]
+    entry_receiver: EntryReceiver,
```

Comment:
So if we don't drop it, I don't we need to pull from it to avoid leaks? May need a separate thread to just pull from this receiver and discard the entries.

Reply:
This is true, the channel buffer could just keep growing unboundedly.

So one solution could be to remove the `?` after `entry_sender.send(consume_queue)?` and instead purposefully ignore that `Result`. But I'm not sure about whether that might cause issues. Should `recv_window` return an error if the `entry_sender.send()` returns an error?